tests/unit/src/main/scala/tests/Library.scala

```diff
@@ -25,95 +25,99 @@ object Library {
-            "0.1.8"
-          ),
-          new Dependency(
-            "com.typesafe.akka",
-            "akka-testkit_2.12",
-            "2.5.19"
-          ),
-          new Dependency(
-            "com.typesafe.akka",
-            "akka-stream_2.12",
-            "2.5.19"
-          ),
-          new Dependency(
-            "com.typesafe.akka",
-            "akka-cluster_2.12",
-            "2.5.19"
-          ),
-          new Dependency(
-            "org.apache.spark",
-            "spark-sql_2.11",
-            "2.2.1"
-          ),
-          new Dependency(
-            "org.eclipse.jetty",
-            "jetty-servlet",
-            "9.3.11.v20160721"
-          ),
-          new Dependency(
-            "org.apache.kafka",
-            "kafka_2.12",
-            "1.0.0"
-          ),
-          new Dependency(
-            "org.apache.flink",
-            "flink-parent",
-            "1.4.1"
-          ),
-          new Dependency(
-            "io.grpc",
-            "grpc-all",
-            "1.10.0"
-          ),
-          new Dependency(
-            "org.scala-lang",
-            "scala-compiler",
-            mtags.BuildInfo.scalaCompilerVersion
-          ),
-          new Dependency(
-            "io.buoyant",
-            "linkerd-core_2.12",
-            "1.4.3"
-          )
+        Dependency.of(
+          "org.scalameta",
+          "scalameta_2.12",
+          "4.1.4"
+        ),
+        Dependency.of(
+          "com.lihaoyi",
+          "scalaparse_2.12",
+          "2.1.0"
+        ),
+        Dependency.of(
+          "com.lihaoyi",
+          "acyclic_2.12",
+          "0.1.8"
+        ),
+        Dependency.of(
+          "com.typesafe.akka",
+          "akka-testkit_2.12",
+          "2.5.19"
+        ),
+        Dependency.of(
+          "com.typesafe.akka",
+          "akka-stream_2.12",
+          "2.5.19"
+        ),
+        Dependency.of(
+          "com.typesafe.akka",
+          "akka-cluster_2.12",
+          "2.5.19"
+        ),
+        Dependency.of(
+          "org.apache.spark",
+          "spark-sql_2.11",
+          "2.2.1"
+        ),
+        Dependency.of(
+          "org.eclipse.jetty",
+          "jetty-servlet",
+          "9.3.11.v20160721"
+        ),
+        Dependency.of(
+          "org.apache.kafka",
+          "kafka_2.12",
+          "1.0.0"
+        ),
+        Dependency.of(
+          "org.apache.flink",
+          "flink-parent",
+          "1.4.1"
+        ),
+        Dependency.of(
+          "io.grpc",
+          "grpc-all",
+          "1.10.0"
+        ),
+        Dependency.of(
+          "org.scala-lang",
+          "scala-compiler",
+          mtags.BuildInfo.scalaCompilerVersion
+        ),
+        Dependency.of(
+          "io.buoyant",
+          "linkerd-core_2.12",
+          "1.4.3"
         )
       )
-      .withClassifiers(List("sources", "_"))
-    val jars = CoursierSmall.fetch(settings)
+      .withClassifiers(Set("sources", "_").asJava)
```

Comment:
iirc, the `"_"` classifier does not work with coursier/interface. You need to call `fetch` twice, once with `"sources"` classifier and one without custom classifiers (to get classfiles)